GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -37,8 +37,11 @@ gdjs.RuntimeGameRenderer = gdjs.RuntimeGamePixiRenderer; //Register the class to
 gdjs.RuntimeGamePixiRenderer.prototype.createStandardCanvas = function(canvasArea) {
     this._canvasArea = canvasArea;
 
+    //This prevents flickering on some mobile devices
+    PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;
+
     //Create the renderer and setup the rendering area
-    this._pixiRenderer = PIXI.autoDetectRenderer(this._game.getDefaultWidth(), this._game.getDefaultHeight());
+    this._pixiRenderer = PIXI.autoDetectRenderer(width=this._game.getDefaultWidth(), height=this._game.getDefaultHeight(), {transparent: true} );
```

Comment:
I'm not sure `transparent: true` is needed? :)
Also `width=...` and `height=...` are not needed. JavaScript do not have named argument, so adding this will in fact affect the global variables width and height in the global scope, which can be confusing and even dangerous (using global variables is highly discouraged and should almost never be needed).
Can you remove the modifications to this line, unless there is some usage that I've not seen? 😉

Reply:
`transparent=true` is from @Bouh 's original suggestion that overcame the flickering by setting the parameter (see referenced bug report #463 .
I don't have an Android phone affected by the flickering so it would be nice if @Bouh could test it with the line removed.

Reply:
`PIXI.glCore.VertexArrayObject.FORCE_NATIVE = true;`
Fix nothing on my phone (Huawei ALE-L21, Android 6.0, EMUI 4.0.3)

`transparent: true` Fix the problem for me
If you have a smartphone try : http://witly.fr/MonumentQuizz/

Reply:
Ok good to know, I happen to miss this detail in the original issue!
@Wend1go Can you add a comment telling that "`transparent: true` is needed to avoid flickering issues on some mobile phones (see https://github.com/4ian/GD/pull/566)"?
Let's keep this but remove "width=..." and "height=..."